scripts/abi_artifact_generator/templates/test_list.txt.jinja_template

```diff
@@ -0,0 +1,4 @@
+/*{- for name, arguments in argument_functions.items() -}*//*( name )*/ /*{ endfor -}*/
+/*{- for name, arguments in return_value_functions.items() -}*//*( name )*/ /*{ endfor }*/
+randomized_state printable_location
+/*( lfsr_seed )*/ /*( iteration_count )*/ /*( stack_byte_count )*/ /*( generated_byte_count )*/
```

Comment:
What is this for?

Reply:
This is the trick I used to avoid having to templatize `runner.c`. It passes all the extra data it needs as part of the header it takes as the main input.